ui-ngx/src/app/modules/home/components/profile/alarm/device-profile-alarm.component.html

```diff
@@ -73,7 +73,6 @@
                 <mat-icon matChipRemove>close</mat-icon>
               </mat-chip-row>
               <input matInput type="text" placeholder="{{'device-profile.alarm-rule-relation-types-list' | translate}}"
-                     style="max-width: 200px;"
```

Comment:
Why do you need to correct styles if translations are corrected in PR?

Reply:
max-width cuts off the last letter (same as any letter that makes the placeholder longer than 200px)